rados/ioctx.go

```diff
@@ -302,22 +303,36 @@ type ObjectListFunc func(oid string)
 // RadosAllNamespaces before calling this function to return objects from all
 // namespaces
 func (ioctx *IOContext) ListObjects(listFn ObjectListFunc) error {
-	var ctx C.rados_list_ctx_t
-	ret := C.rados_nobjects_list_open(ioctx.ioctx, &ctx)
-	if ret < 0 {
-		return getError(ret)
+	page_results := C.size_t(1000)
```

Comment:
Since this is a "magic" value. Let's make it a const at the file level. Name it something like `defaultListObjectsResultSize`.